drivers/can/can_mcan.c

```diff
@@ -23,6 +23,22 @@ LOG_MODULE_DECLARE(can_driver, CONFIG_CAN_LOG_LEVEL);
 #define MCAN_MAX_DLC CAN_MAX_DLC
 #endif
 
+#if CONFIG_HAS_CMSIS_CORE_M
+#include <arch/arm/aarch32/cortex_m/cmsis.h>
+
+#if __DCACHE_PRESENT == 1
+#define CACHE_INVALIDATE(add, size) SCB_InvalidateDCache_by_Addr((addr), (size))
```

Comment:
This isn't compiling as-is, is the `add` in this macro definition meant to be `addr`?

Reply:
Yep. Sorry, forgot to push again. I tried to compile on top of your PR and actually fixed this locally.